### PR TITLE
Fix #1545 : Changed styles in dialog_basic_theme.xml and updated SettingsAct…

### LINKED ISSUE #1545: No proper highlighing in theme changing option
**Actual Behaviour**

The theme changing menu does not highlight the present option selected and does not highlight while changing the theme.

**Expected Behaviour**

The option selected in the menu should be properly highlighted while selecting and also after selection.

**Steps to reproduce it**

Open the settings area and scroll down to the option to change the theme and select the theme.

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by ParthPratim**
…ivity.java

Fix #1545 

Changes: Changed the UI for dialog theme and added proper highlighting of option
screenshot_2017-12-31-14-12-57-190

Screenshots for the change:

### CODE REVIEW

File: app/src/main/java/org/fossasia/phimpme/gallery/activities/SettingsActivity.java [hunk 1]
```diff
@@ -492,35 +492,47 @@ private void baseThemeDialog(){
...
-        darkAmoledSelect.setIcon("gmd-done");
+        darkAmoledSelect.setIcon("gmd-done");*/
 
         switch (getBaseTheme()){
             case ThemeHelper.LIGHT_THEME:
                 whiteSelect.setVisibility(View.VISIBLE);
                 darkSelect.setVisibility(View.GONE);
                 darkAmoledSelect.setVisibility(View.GONE);
+                themeTextWhite.setTextColor(R.color.md_light_primary_text);
+                themeTextDark.setTextColor(R.color.md_light_primary_text);
```
Comment (mayank8318) [inline app/src/main/java/org/fossasia/phimpme/gallery/activities/SettingsActivity.java]: Use ContextCompat please.

File: app/src/main/java/org/fossasia/phimpme/gallery/activities/SettingsActivity.java [hunk 2]
```diff
@@ -492,35 +492,47 @@ private void baseThemeDialog(){
...
+                themeTextDarkAmoled.setTextColor(R.color.md_light_primary_text);
                 break;
             case ThemeHelper.DARK_THEME:
                 whiteSelect.setVisibility(View.GONE);
                 darkSelect.setVisibility(View.VISIBLE);
                 darkAmoledSelect.setVisibility(View.GONE);
+                themeTextWhite.setTextColor(getResources().getColor(R.color.white));
+                themeTextDark.setTextColor(getResources().getColor(R.color.white));
```
Comment (mayank8318) [inline app/src/main/java/org/fossasia/phimpme/gallery/activities/SettingsActivity.java]: Use ContextCompat please.

File: app/src/main/java/org/fossasia/phimpme/gallery/activities/SettingsActivity.java [hunk 3]
```diff
@@ -492,35 +492,47 @@ private void baseThemeDialog(){
...
         final IconicsImageView darkAmoledSelect = (IconicsImageView) dialogLayout.findViewById(R.id.dark_amoled_basic_theme_select);
 
-        themeIconWhite.setIcon("gmd-invert-colors");
+        /*themeIconWhite.setIcon("gmd-invert-colors");
         themeIconDark.setIcon("gmd-invert-colors");
         themeIconDarkAmoled.setIcon("gmd-invert-colors");
         whiteSelect.setIcon("gmd-done");
         darkSelect.setIcon("gmd-done");
-        darkAmoledSelect.setIcon("gmd-done");
+        darkAmoledSelect.setIcon("gmd-done");*/
```
Comment (anantprsd5) [inline app/src/main/java/org/fossasia/phimpme/gallery/activities/SettingsActivity.java]: Remove the commented lines.

**Comment by isuruAb** [comment]
Closed this because the user no longer actively updating or working on this issue and PR.

**Pull request closed without merging**